File: python/logic2/lone_sum.py
#codingbat link: https://codingbat.com/prob/p143951

def lone_sum(a, b, c):
    if(a == b):
        if(c == a):
            return 0
        else:
            return c
    elif(b == c):
        if(a == c):
            return 0
        else:
            return a
    elif(a == c):
        if(b == a):
            return 0
        else:
            return b
    else:
        return a + b + c

File: java_code/logic2/loneSum.java
package java_code.logic2;

//codingbat link: https://codingbat.com/prob/p148972

public class loneSum {

    public int loneSum(int a, int b, int c){
        int finalSum = a + b + c;

        if(a == b && a == c) finalSum = 0;
        else if(a == c) finalSum = finalSum - (a + c);
        else if(a == b) finalSum = finalSum - (a + b);
        else if(c == b) finalSum = finalSum - (c + b);

        return finalSum;
    }
    
}
